Rakefile

```diff
@@ -1,10 +1,14 @@
 require 'github_changelog_generator/task'
+require 'dependency_checker/version'
+require 'rspec/core/rake_task'
 
 GitHubChangelogGenerator::RakeTask.new :changelog do |config|
+  version = DependencyChecker::VERSION
```

Comment:
The ruby changelog generator is no longer used with this set of workflows so this config can probably be removed.

Reply:
Updated